projects/go-style-guide/src/app/features/ui-kit/components/progress-bar-docs/progress-bar-docs.component.html

```diff
@@ -0,0 +1,139 @@
+<header class="go-page-title">
+  <h1 class="go-heading-1">{{ pageTitle }}</h1>
+  <a [href]="linkToSource" target="_blank">
+    <go-button buttonVariant="neutral"> View on GitHub </go-button>
```

Comment:
Is this page only visibly internally? Just wondering because there are a lot of hard coded strings/text in English on this page... should they be translated? If this page is internal only then translation are not necessary..